recipe/meta.yaml

```
@@ -68,7 +69,7 @@ outputs:
         - {{ pin_subpackage('libxgboost', exact=True) }}
       run:
         - {{ pin_subpackage('libxgboost', exact=True) }}
-        - {{ pin_subpackage('_py-xgboost-mutex', exact=True) }}
+        - _py-xgboost-mutex
```

```suggestion
        - _py-xgboost-mutex 2.0
```
To stay in line with the previous pin.